### PR TITLE
add release 1-31-0 blog post draft

### PR DESCRIPTION
**Pull request by solvingj**

### CODE REVIEW

File: _posts/2020-11-03-New-conan-release-1-31.markdown [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+the use of CMake, but plan to explore the alternatives in the future after we
+have some feedback on these preliminary implementations. Cross-building for
+these mobile platforms is one of the major use-cases for Conan, so we believe
+this step is pretty significant in maturing the toolchain abstraction.
+
+## New MsBuildCmd Helper
+
+In the previous release, we've included a POC of the new `MsBuildToolchain`.
+This was part of the effort to restructure the way we interact with `MsBuild`.
+This release contains another part of that effort which is the new `MsBuildCmd`
```
Comment (memsharded) [inline _posts/2020-11-03-New-conan-release-1-31.markdown:84]: It is not called ``MsBuildCmd``, it is the same ``MSBuild()`` name, but internally it is a new class. This is an attempt to help to migrate to toolchains without changing this name too.

**Review by memsharded — CHANGES_REQUESTED** [review]
Please put tomorrow Thursday 5th date

File: _posts/2020-11-03-New-conan-release-1-31.markdown [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+Conan 1.31.0 features a few more significant steps toward our new toolchain
+strategy, coupled with a lengthy list of quality-of-life features and fixes.
+
+## Upload Performance Improvements
+
+This release includes a long-overdue change, which represents an optimization
+which is so impactful that it effectively amounts to a bug fix for some
+use cases. The change is such that the Conan client will now keep the
+`conan_export.tgz` and `conan_source.tgz` in the local cache after it is done
```
[the remark this answers is not in the document]
Comment (solvingj) [reply _posts/2020-11-03-New-conan-release-1-31.markdown:16]: Ok, fixing, didn't realize tat.

File: _posts/2020-11-03-New-conan-release-1-31.markdown (hunk 1 above)
Comment (solvingj) [reply _posts/2020-11-03-New-conan-release-1-31.markdown:84]: Ok, I almost asked about it but figured we would address it today.

File: _posts/2020-11-05-New-conan-release-1-31.markdown [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+## Upload Performance Improvements
+
+This release includes a long-overdue change, which represents an optimization
+which is so impactful that it effectively amounts to a bug fix for some use
+cases. The change is such that the Conan client will now keep the
+`conan_export.tgz`, `conan_source.tgz`, and `conan_package.tgz` in the local
+cache after it is done extracting the contents. Previously, these files were
+deleted once extracted.
+
+How is this an optimization? Well buckle up because this is a long and sordid
```
Comment (danimtb) [inline _posts/2020-11-05-New-conan-release-1-31.markdown:20]: I feel this whole explanation about this feature regarding compression and decompression is a bit lengthy and I believe the user will be more interested in the consequences than in the whole explanation. However, reading the whole blogpost, it fits with the following explanations, and, as we don't have much to talk about, I am fine with it.

**Review by danimtb — APPROVED** [review]

**Pull request merged**